spec/controllers/application_controller_spec.rb

```diff
@@ -1,6 +1,10 @@
 require 'ostruct'
 
 describe ApplicationController do
+  it ".model" do
```

Comment:
Being picky, sorry: Can't handle the use of `it` without it reading right. Either describe "#model" do; it "returns the model" do; ennd;` or even just `example "#model"` instead of `it`. :pray: